locales/en-US/server.ftl

```diff
@@ -11,10 +11,13 @@ gSettings = Settings
 gSignIn = Sign In
 
 ## Header
-signInButton =
-    .aria-label = Sign In
 settingsButton =
```

Comment:
If you change attributes in an existing string, you need to change the ID of the string.

Reply:
Done